pkg/features/kube_features.go

```diff
@@ -974,7 +974,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	JobPodReplacementPolicy: {Default: false, PreRelease: featuregate.Alpha},
 
-	JobReadyPods: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
oh, add a comment for when the feature gate should be removed: in 1.31

See how others do it below

Reply:
added https://github.com/kubernetes/kubernetes/pull/121302/commits/e2c75cfdf36c955ffe73a58a4c75870b7017ddc2